src/reanimated2/layoutReanimation/web/animationsManager.ts

```diff
@@ -89,9 +89,15 @@ function tryGetAnimationConfigWithTransform<
     return null;
   }
 
+  type ConstructorWithStaticContext = {
```

Comment:
This should use imported `LayoutAnimationStaticContext`.

Reply:
Both done in [fd57b63](https://github.com/software-mansion/react-native-reanimated/pull/5562/commits/fd57b63bbddc9d9a55fe555a828f6040b3814072).